src/main/webapp/css/timeline.css

```diff
@@ -52,6 +52,14 @@
   border-radius: 0.5em;
 }
 
+.locationLink {
+  color: grey;
```

Comment:
My suggestion is include something like `text-decoration: underline` here as well so that even though the text is in grey (which looks better) people still know its  a link that is clickable.

Reply:
okay! thanks for your suggestion!